en/sensor/bitcraze_flow.md

```diff
@@ -0,0 +1,67 @@
+# Bitcraze Optical Flow Board
+
+
```

Comment:
A few words about what is special about this board would be good. Ie why would you select this over an alternative.

If not, just an intro of the key specs. Like for PX4FLow I had:

> PX4FLOW is an optical flow smart camera. It has a native resolution of 752x480 pixels and calculates optical flow on a 4x binned and cropped area at 400 Hz, giving it a very high light sensitivity.

These words can then be used in the parent "Optical Flow" doc to refer to this particular option.